File: Porker.cs
namespace KKN_Nagrak_RT04
{
    public class Proker
    {
        public string NamaProgram { get; set; }
        public string Bidang { get; set; }
        public string Status { get; set; } // "Belum Mulai", "Progres", "Selesai"

        public Proker(string namaProgram, string bidang, string status)
        {
            NamaProgram = namaProgram;
            Bidang = bidang;
            Status = status;
        }
    }
}

File: Porker.java
public class Proker {
    private String namaProgram;
    private String bidang;
    private String status; // "Belum Mulai", "Sedang Berjalan", "Selesai"

    public Proker(String namaProgram, String bidang, String status) {
        this.namaProgram = namaProgram;
        this.bidang = bidang;
        this.status = status;
    }

    public String getNamaProgram() { return namaProgram; }
    public String getBidang() { return bidang; }
    public String getStatus() { return status; }
    
    public void setStatus(String status) { this.status = status; }
}
